### PR TITLE
Switch service

### PR DESCRIPTION
**Pull request by pelikhan**

### CODE REVIEW

File: services/switch.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Indicates whether the switch is currently active (on).
+
+    enum Variant: u32 {
+        Slide = 1,
+        Tilt = 2,
+        PushButton = 3,
+        Tactile = 4,
+        Toggle = 5,
+        Light = 6
```
Comment (mmoskal) [inline services/switch.md:20]: what's Light?

File: services/switch.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    const variant?: Variant @ variant
+
+Describes the type of switch used.
+
+    const auto_off_delay?: u16.16 s @ 0x180
+
+Specifies the delay without activity to automatically turn off after turning on.
```
Comment (mmoskal) [inline services/switch.md:28]: is this to specify mechanical properties of the switch?

Comment (pelikhan) [reply services/switch.md:28]: It's not uncommon for light switches to have a auto-off feature built-in. For example, light switches in staircases.

Comment (mmoskal) [reply services/switch.md:28]: clarified

File: services/switch.md (hunk 1 above)
Comment (mmoskal) [reply services/switch.md:20]: renamed

**Pull request merged**